stdlib/2/__builtin__.pyi

```diff
@@ -939,10 +939,10 @@ class UnicodeWarning(Warning): ...
 class BytesWarning(Warning): ...
 class ResourceWarning(Warning): ...
 
-def eval(s: str, globals: Dict[str, Any] = ..., locals: Dict[str, Any] = ...) -> Any: ...
+def eval(s: Union[str, unicode], globals: Dict[Union[str, unicode], Any] = ..., locals: Dict[Union[str, unicode], Any] = ...) -> Any: ...
```

Comment:
I believe the dict ones will cause trouble because Dict is invariant in its key type, so now mypy will *only* let you pass a dictionary whose keys are `Union[str, unicode]`. Mapping is not correct here either, because eval and exec will modify these dictionaries.